server/views/pages/overview/partials/goalsTab/goalsTabContents.njk

```diff
@@ -51,7 +51,25 @@
             <p class="govuk-body" data-qa="no-in-progress-goals-message">{{ prisonerSummary.firstName }} {{ prisonerSummary.lastName }} has no goals in progress.</p>
           {% endif %}
         {% endset %}
-
+        {% if featureToggles.completedGoalsEnabled %} 
+          {% set completedGoalsHtml %}
+            <h2 class="govuk-heading-m">Completed goals</h2>
+            {% if completedGoals.length > 0 %}
+              {% for goal in completedGoals | sort(attribute = 'updatedAt', reverse = true) %}
+                {{ goalSummaryCard({
+                          goal: goal,
+                          attributes: {
+                            'data-qa': 'completed-goal-summary-card'
+                          },
+                          classes: 'archived-or-completed-goal-summary-card',
```

Comment:
Whats the classname "archived-**or-completed**-goal-summary-card"?
Does that classname already exist? What is it for?

I guess I'm just a bit confused by the archived-**or-completed** bit 🤔  - surely these ones are the completed ones, and the ones below are the archived ones? 🤔